Puppetfile

```diff
@@ -3,13 +3,10 @@ forge 'https://forgeapi.puppetlabs.com'
 mod 'katello/common',                '>= 1.0.1 < 1.1.0'
 mod 'katello/service_wait',          '>= 1.0.3 < 1.1.0'
 
-mod 'katello/certs',                 '>= 3.0.0 < 3.1.0'
-mod 'katello/qpid',                  '>= 3.1.0 < 3.2.0'
-mod 'katello/pulp',                  '>= 5.0.0 < 5.1.0'
-mod 'katello/candlepin',             '>= 4.0.0 < 4.1.0'
+mod 'katello/certs',                 '>= 3.0.1 < 3.0.2'
+mod 'katello/qpid',                  '>= 4.0.0 < 4.1.0'
+mod 'katello/pulp',                  '>= 5.1.0 < 5.2.0'
+mod 'katello/candlepin',             '>= 5.0.0 < 5.1.0'
 
-mod 'katello/foreman_proxy_content', '>= 6.0.0 < 6.1.0'
+mod 'katello/foreman_proxy_content', '>= 7.0.0 < 7.1.0'
 mod 'katello/katello',               '>= 7.0.0 < 7.1.0'
-
-# Katello devel specific modules
-mod 'katello/katello_devel',         :git => 'https://github.com/Katello/puppet-katello_devel'
```

Comment:
If we drop this here, you also need some other changes like in https://github.com/Katello/katello-installer/commit/9fbd7a17ad899708193161c8969141971c363d92 (drop answers, remove it from tests, Rakefile)